www/src/pages/en/folder-structure.mdx

```diff
@@ -47,7 +47,7 @@ The `pages` folder contains all the pages of the Next.js application. The `index
 
 #### `src/pages/api`
 
-The `api` folder contains all the API routes of the Next.js application. The `examples.ts` file contains an example of a route that uses the [Next.js API route](https://nextjs.org/docs/api-routes/introduction) feature along with Prisma. The `restricted.ts` file contains an example of a route that uses the [Next.js API route](https://nextjs.org/docs/api-routes/introduction) feature and is protected by [NextAuth.js](https://next-auth.js.org/).
+The `api` folder contains all the API routes of the Next.js application.
```

Comment:
I think we should put a few more words here given that it's one of Next.js' strong suits

Reply:
linking here probably makes sense https://nextjs.org/docs/api-routes/introduction

Reply:
I added the link to the Next.js api routes docs, thank you for the suggestion!